app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
@cankarm I see you concern. For now, I think that MiQ terminology should be used wherever possible, but I am not target audience.

Further to @agrare's question starting this thread: Do you have an idea if it's possible to link the template VM image with the actual instance? I couldn't find anything in fog nor raw requests to vcloud, but would like to double check this with you.